### PR TITLE
Set minimum ttl to zero to fix issue #1320

### PR DESCRIPTION
**Pull request by MatthiasGabriel**
Add minimum ttl to ios traceroute command.
Fixes Issue #1320

### CODE REVIEW

File: napalm/ios/ios.py [hunk 1]
```diff
@@ -3284,7 +3284,7 @@ def traceroute(
             command += " source {}".format(source)
         if ttl:
             if isinstance(ttl, int) and 0 <= ttl <= 255:
-                command += " ttl {}".format(str(ttl))
+                command += " ttl 0 {}".format(str(ttl))
```
Comment (mirceaulinic) [inline napalm/ios/ios.py:3287]: I'm not sure this syntax is supported on every IOS version. Did you give it a try @MatthiasGabriel?

Comment (MatthiasGabriel) [reply napalm/ios/ios.py:3287]: Any suggestions which versions I should check?
Also as now mentioned in the issue #1320 I'm not sure about the value 0 either. In my opinion 1 seems like the "default".

Comment (mirceaulinic) [reply napalm/ios/ios.py:3287]: Gah, not sure what to say, I fortunately don't have a Cisco IOS to check this out. Perhaps @ktbyers would have more input on this matter? I'd also suggest you to engage with the community and see what other IOS users think about it.

Comment (MatthiasGabriel) [reply napalm/ios/ios.py:3287]: Thanks for your feedback @mirceaulinic 
During the day I added some new findings to the issue #1320

**Review by mirceaulinic — APPROVED** [review]

**Pull request merged**